src/mark.ts

```diff
@@ -119,12 +120,19 @@ export interface MarkConfig extends VgMarkConfig {
   /**
    * Whether the mark's color should be used as fill color instead of stroke color.
    * All marks except "point", "line", and "rule" are filled by default.
+   *
+   * __Default value:__ `true` for all marks except `point` and `false` for `point`.
```

Comment:
Remove the redundant sentence "All marks except "point", "line", and "rule" are filled by default." above.